tests/ia/pages/alertsPage.js

```diff
@@ -75,4 +83,28 @@ module.exports = {
       );
     }
   },
+
+  async checkAllAlertsColor(expectedStates) {
+    const alertsStates = await I.grabTextFromAll('//td[3]');
+    const criticalColor = await I.grabCssPropertyFrom(this.elements.criticalSeverity, 'color');
+    const highColor = await I.grabCssPropertyFrom(this.elements.highSeverity, 'color');
+    const noticeColor = await I.grabCssPropertyFrom(this.elements.noticeSeverity, 'color');
+    const warningColor = await I.grabCssPropertyFrom(this.elements.warningSeverity, 'color');
+
+    for (const i in alertsStates) {
+      assert.equal(alertsStates[i], expectedStates, `Alert has not ${expectedStates} state`);
+    }
+
+    if (expectedStates === 'Silenced') {
+      assert.notEqual(criticalColor, this.colors.critical, 'Critical alert is unsilence');
```

Comment:
since you are checking for exact color please use `assert.equal` too here

Reply:
ok.add silence color